hawk/modules/websocket.js

```diff
@@ -0,0 +1,64 @@
+const WebSocket = require('ws');
```

Comment:
We have a more clear implementation in Hawk JavaScript 
https://github.com/codex-team/hawk.javascript/blob/master/src/modules/websocket.js